.gitignore

```diff
@@ -24,3 +24,4 @@
 *.ji
 
 .DS_Store
+get_toolchain.log
```

Comment:
I agree. I can not for the life of me figure out how to remove that file from the pull request (or requests as it seems to have combined both). I really just want the changes to `linalg/generics.jl` and `tests/linalg/generics.jl`. But man every time I try to follow the guides on rebasing or amending ... it nukes everything. Hardest 10 line change to code I have had to do. I am might be to dumb to use git :(